Detections/MultipleDataSources/ExtensiveAWSComputeResurseDeployements.yaml

```diff
@@ -0,0 +1,54 @@
+id: 9e457dc4-81f0-4d25-bc37-a5fa4a17946a
+name: Extensive AWS Compute Resurse Deployements

```

Comment:
Suggest renaming to "Suspicious AWS EC2 Compute Resource Deployments"

Reply:
done